src/Symfony/Component/Notifier/Bridge/Smsapi/SmsapiTransport.php

```diff
@@ -65,10 +65,10 @@ protected function doSend(MessageInterface $message): SentMessage
             ],
         ]);
 
-        if (200 !== $response->getStatusCode()) {
-            $error = $response->toArray(false);
+        $responseContent = $response->toArray(false);
 
-            throw new TransportException(sprintf('Unable to send the SMS: "%s".', $error['message']), $response);
+        if ((isset($responseContent['error']) && null !== $responseContent['error']) || (200 !== $response->getStatusCode())) {
```

Comment:
`isset($foo['bar'])` returns also true when the key exists but it's value is null.
You can remove the second assertion

Reply:
I need to confirm if the value is not null. Checking only if the key exists is not sufficient